flink-connectors/flink-connector-base/src/test/java/org/apache/flink/connector/base/source/reader/CoordinatedSourceRescaleITCase.java

```diff
@@ -72,7 +72,7 @@ private File generateCheckpoint(File checkpointDir, int p) throws IOException {
             env.execute("create checkpoint");
             throw new AssertionError("No checkpoint");
         } catch (Exception e) {
-            assertEquals(CREATED_CHECKPOINT, ExceptionUtils.getRootCause(e).getMessage());
+            assertTrue(findThrowableWithMessage(e, CREATED_CHECKPOINT).isPresent());
```

Comment:
I'd suggest to use `assertThat(e, FlinkMatchers.containsMessage(CREATED_CHECKPOINT));`